contracts/staking/TokenStaking.sol

```diff
@@ -1114,17 +1128,17 @@ contract TokenStaking is Initializable, IStaking, Checkpoints {
         nuInTStake = operatorStruct.nuInTStake;
     }
 
-    /// @notice Returns start staking timestamp for T stake.
+    /// @notice Returns start staking timestamp for T/NU stake.
     /// @dev    This value is set at most once, and only when a stake is created
-    ///         with T tokens. If a stake is created from a legacy stake,
-    ///         this value will remain as zero
-    function getStartTStakingTimestamp(address operator)
+    ///         with T or NU tokens. If a stake is created from a legacy KEEP
+    ///         stake, this value will remain as zero
+    function getStartStakingTimestamp(address operator)
```

Comment:
Why don't we set this value for `stakeKeep` as well? 

We do not have to use it for undelegation validation given it does not make sense for KEEP stakers but if `getStartStakingTimestamp` works only for liquid T and NU, the function name is confusing. 

We could also use `getStartStakingTimestamp` in the dApp to display when someone started staking and if they staked, without having to scan events.

Reply:
Yes, we could do that. Just tried to minimize changes

Reply:
See #73